rpm-oxide.spec.in

```diff
@@ -29,7 +29,7 @@ BuildRequires:  rpm-devel >= 4.14.2.1-4
 BuildRequires:  rust-debugger-common
 BuildRequires:  rust-cc-devel
 BuildRequires:  rust-libc-devel
-%define qubes_cargo RUSTFLAGS='-Cdebuginfo=2 -Clink-arg=-Wl,-z,relro,-z,now' \\\
+%define qubes_cargo RUSTFLAGS='-Cstrip=none -Cdebuginfo=2 -Clink-arg=-Wl,-z,relro,-z,now' \\\
```

Comment:
Maybe better avoid setting custom RUSTFLAGS at all?
And add `%set_build_flags` at the start of `%build` (and `%check`?) to have default ones set

Reply:
@DemiMarie what's the original reason to not use default Fedora flags?

Reply:
I don’t remember.  I suspect it dates back to me trying to support the ancient Rust in Fedora 25 for R4.0.

Reply:
@fepitre so, in this case can you try removing custom RUSTFLAGS?